examples/pdc_global_component_contact.rb

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby
+
+require './lib/pdc'
+
+def main
+  PDC.configure do |config|
+    config.site = 'https://pdc.engineering.redhat.com'
```

Comment:
Not that big of a deal, but since it's an upstream project, I'd use the Fedora PDC here.

Reply:
I agree @romanofski, however I think this belongs in its own PR, as the token fetch for this gem is set-up for this url.

Reply:
@jacobseth yes, I think so, it's better to keep this url.